frontend/src/metabase/entities/databases/forms.js

```diff
@@ -288,6 +288,13 @@ const forms = {
         description: t`By default, Metabase does a lightweight hourly sync and an intensive daily scan of field values. If you have a large database, we recommend turning this on and reviewing when and how often the field value scans happen.`,
         hidden: !engine,
       },
+      {
+        name: "refingerprint",
+        type: "boolean",
+        title: t`Periodically refingerprint tables`,
```

Comment:
I don't think we use the term "fingerprint" or "fingerprinting" in any userfacing instances currently, so I'd suggest something of a more universal term.

```suggestion
        title: t`Periodically reanalyze fields`,
```